WooCommerce/WooCommerceTests/POS/ViewModels/TotalsViewModelTests.swift

```diff
@@ -0,0 +1,15 @@
+import XCTest
+
+final class TotalsViewModelTests: XCTestCase {
```

Comment:
@iamgabrielma 👋 since the PR is already pretty big, maybe we can add the test cases in a separate PR?

Reply:
That sounds good, I've opened a separate issue so we can handle those: https://github.com/woocommerce/woocommerce-ios/issues/13217